src/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -3402,4 +3402,51 @@
   <data name="FailLoadAssemblyUnderPregenMode" xml:space="preserve">
     <value>"Fail to load assembly {0} or {0} doesn't exist under PreGen Mode.</value>
   </data>
+  <data name="HelpDescription" xml:space="preserve">
+    <value>Generates serialization code for use with XmlSerializer.
```

Comment:
The tool takes in an assembly and output an assembly. Code generation is just implementation detail, which we should not mention as part of description.